packages/material-ui/src/ObjectFieldTemplate/ObjectFieldTemplate.tsx

```diff
@@ -28,14 +28,14 @@ const ObjectFieldTemplate = ({
         <TitleField id={`${idSchema.$id}-title`} title={title} required={required} />
       )}
       {description && <DescriptionField id={`${idSchema.$id}-description`} description={description} />}
-      <Grid container={true} spacing={2} style={{ marginTop: '10px' }}>
+      <Grid id={idSchema.$id} container={true} spacing={2} style={{ marginTop: '10px' }}>
```

Comment:
Well, the default theme doesn't have these MUI wrapper. => https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/fields/SchemaField.js#L197

Solving it the same way as the core package for example would not help, you have to interact at the wrapper level because of CSS nature. This is why it is done this way in my case

Reply:
<img width="1709" alt="Screenshot 2022-06-17 at 14 38 49" src="https://user-images.githubusercontent.com/8804297/174309708-e127fafc-4094-406b-8aaa-058340903f3b.png">

Reply:
I see, the unique id is there for the main bootstrap-3 theme (https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/fields/ObjectField.js#L15) but not for other themes like fluent, as you mentioned.

Reply:
@benjaminbours can you explain again your use case with css ("allow precise css selector")? Some examples of CSS selectors you'd like to use which would be possible with this PR? We might want to also add them into a test to ensure these selectors continue to work in the future for this theme.